vipps-report-api-faq.md

```diff
@@ -33,6 +34,47 @@ We are still quite early in the planning and design phase and will use the
 draft documentation here to discuss with potential users of the API
 and make sure we are developing the right solution.
 
+## Which API keys give access to the API?
+
+In short: These API keys give access to the Report API:
+* The merchant's own API keys (the same API keys that are used to make payments)
+* Partner keys (the same API keys that are used to make payments)
+* Accounting partners' API keys (after the merchant has given the accounting partner access)
+
+### The merchant's own API keys
+
+The merchant's own API keys give full access to the Report API.
+
+If the merchant uses an integration partner (see
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
+it is the same as using the merchant's own API keys.
+
+When a merchant shares it's API keys for a sale unit with an integration partner,
+Vipps has no way of knowing whether the API calls are made by the merchant or
+the integration partner.
+
+### Partner keys
+
+If the merchant has a sale unit that is configured with a platform partner (see
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
+the platform partner can use
+[partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys)
+to make requests to the Report API.
+
+[Partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys)
+lets a partner make payments on behalf of a merchant, and the same API keys
+also give access to the Report API.
+
+## Specifying a accounting partner
+
+In addition to the above, the merchant may add one or more accounting partners.
+An accounting partner will get access to the Report API, but will not be allowed
+to make payments, or move money in any way. The accounting partner can only see
+reports of the payments that have been made.
```

Comment:
Pretty much same as above, "access to the Report API" might be enough for this part, but if the intended reader is a merchant it could be a bit too vague. But we can definitely just leave it like this for now.

Reply:
Ok, leaving as-is.